### DISCUSSION TITLE
Start new project from existing project

### DISCUSSION THREAD

**Discussion by danosono** [community/community#58136] [Repositories] [answered]
### Clone project

Question

### Fork project

I want to copy my own project to start a completely separate one; new one will not be merged back to the original.

How should I go about it? I prefer Github desktop but can use Gitbash.

Thanks!!!

**Comment by danosono** [comment]
Is it OK to: 

- clone original repo
- delete the .git folder and .gitattributes and .gitignore from the cloned repo folder
- create a new repo on github.com
- connect the new github.com repo to the cloned folder using Github Desktop

?

**Comment by Gustavonn07** [comment, marked as answer]
Well I would make a new repository and copy the files to this new one, using the files already saved on the computer and copying them to the new repository

**Reply by danosono** [reply]
makes sense, thanks! I will do this.
I know it seems simple but I searched for the best way and I get LOTS of Clone/Fork/Branch content that takes me down so many rabbit trails that I get confused.

**Reply by Gustavonn07** [reply]
You're welcome, this fork thing I don't think would be useful, since you're using your own repository.
I hope everything went well (if so, you could mark it as answered marking my comement so that other people already know that your question worked)

**Reply by danosono** [reply]
I took your advice and then followed this: https://github.com/mikewesthad/unity-git-and-lfs
This was also helpful: https://www.youtube.com/watch?v=sjlVox4f5Ww

Thanks for taking the time to answer my simple question; it really did help! I just needed a push.

One problem I ran into was that the project I was cloning already had its own (my) repo and the new one (copy) kept wanting to see the old project as its head. I had to go create a new repo on Github.com and then activate that on Github desktop (there is an option for this on github.com after making a new repo). After that, I placed the old project files in the new repo folder (both on my PC). 

Now I have two completely separate projects so I can veer off with the new one (make it a mobile game) and it can push to its own repo. I was worried I would foul my working project that is published and receives updates.

Thanks again!!
